src/test/java/com/impactupgrade/nucleus/it/StripeToSfdcIT.java

```diff
@@ -273,6 +274,69 @@ public void coreSubscription() throws Exception {
     clearSfdc(customer.getName());
   }
 
+  @Test
+  public void ignoreStripeMetadataCampaignIdIfRDCampaignUpdatedInSFDC() throws Exception {
```

Comment:
Great to have the IT test in place - good move!

Reply:
Ya man, trying to get into a better habit...